src/extension/background-script/db.ts

```diff
@@ -43,25 +52,32 @@ class DB extends Dexie {
       payments:
         "++id,allowanceId,host,location,name,description,totalAmount,totalFees,preimage,paymentRequest,paymentHash,destination,createdAt",
     });
+    this.version(2).stores({
+      blocklist: "++id,host,name,imageURL,isBlocked,createdAt",
+    });
     this.on("ready", this.loadFromStorage.bind(this));
     this.allowances = this.table("allowances");
     this.payments = this.table("payments");
+    this.blocklist = this.table("blocklist");
   }
 
   async saveToStorage() {
     const allowanceArray = await this.allowances.toArray();
     const paymentsArray = await this.payments.toArray();
+    const blocklistArray = await this.blocklist.toArray();
     await browser.storage.local.set({
       allowances: allowanceArray,
       payments: paymentsArray,
+      blocklist: blocklistArray,
     });
     return true;
   }
 
+  // TODO: DRY up the duplication here.
```

Comment:
For who is this comment? We usually create an issue for TODOs and link it here. Otherwise it's hard to forget about it and might never be done.

Reply:
it was me. wasn't sure if it hurts enough to create an issue 🙈

Reply:
If it doesn't hurt enough we do not need it? :P